src/Workspaces/Core/Desktop/project.json

```diff
@@ -1,6 +1,10 @@
 {
   "dependencies": {
     "ManagedEsent": "1.9.4",
+    "SQLitePCLRaw.core": "1.1.2",
+    "SQLitePCLRaw.bundle_green": "1.1.2",
+    "SQLitePCLRaw.lib.e_sqlite3.v110_xp": "1.1.2",
+    "SQLitePCLRaw.provider.e_sqlite3.net45": "1.1.2",
```

Comment:
Authoring is not part of this PR.  I've just done the authorin to get me off the ground.  I need you and @jasonmalinowski To let me know how to actually author this stuff.  Note that the nuget packages here are very weird as they want to go update your msbuild targets and whatnot.  I have no idea how you're supposed to do that in a project.json world.